src/components/CheckboxGroup.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { CheckboxGroupField, Checkbox } from '@dhis2/ui-core'
+
+import { FieldAdapter, adapterComponentProps } from './FieldAdapter.js'
+import { useToggleGroupChangeHandler } from './shared/hooks.js'
+import { toggleGroupOptionsProp } from './shared/propTypes.js'
+
+const CheckboxGroupComponent = ({ options, value, onChange, ...rest }) => {
+    const handleChange = useToggleGroupChangeHandler(value, onChange)
+
+    return (
+        <CheckboxGroupField
+            {...rest}
+            value={value || []}
```

Comment:
Shouldn't we just rely on the user setting final-form's `defaultValue` prop on the `Field`? That would make this component simpler, and I think for me less confusing as a user. Otherwise you might be wondering where this default came from, and would have to look at ui-forms source to find out.

It would require a tiny bit more work, but make things more explicit and less magic.

Reply:
I would like to end up with a set of components that work in their default setup out-of-the-box. Always having to set a `defaultValue` would break this principle, so I would prefer not to do that.

And regarding the magic: we are dealing with that anyway. RFF "does a lot of stuff", so making by making our components more transparent there will still be magic to deal with...